pkg/custom_detectors/custom_detectors.go

```diff
@@ -166,14 +169,45 @@ func (c *CustomRegexWebhook) createResults(ctx context.Context, match map[string
 			}
 			req.Header.Add(key, strings.TrimLeft(value, "\t\n\v\f\r "))
 		}
-		res, err := httpClient.Do(req)
+		resp, err := httpClient.Do(req)
 		if err != nil {
 			continue
 		}
-		// TODO: Read response body.
-		res.Body.Close()
-		if res.StatusCode == http.StatusOK {
+		defer func() {
+			_, _ = io.Copy(io.Discard, resp.Body)
+			_ = resp.Body.Close()
+		}()
+
+		if resp.StatusCode == http.StatusOK {
+			// mark the result as verified
 			result.Verified = true
+
+			// read the Content-Type header and response body
+			respContentType := resp.Header.Get("Content-Type")
+			body, err := io.ReadAll(resp.Body)
```

Comment:
@mcastorina I could not recall where we had a discussion about PII. I remember, we decided not to include PII in extraData. Will this needed to be care here as well ?

Reply:
That's a great point. Custom detector servers are user supplied (or at least user configured), so I don't *think* PII would be an issue here.. @zricethezav what are your thoughts?